modules/tinymce/CHANGELOG.md

```diff
@@ -60,6 +60,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `media` plugin no longer treats `iframe`, `video`, `audio` or `object` elements as "special" and will validate the contents against the schema #TINY-8382
 - Renamed the `getShortEndedElements` Schema API to `getVoidElements` #TINY-8344
 - Changed the default statusbar element path delimiter from `»` to `›` #TINY-8372
+- Changed the default tag for the strikethrough format to the `s` tag when using a html 5 schema #TINY-8262
```

Comment:
It also converts `strike` to `s` elements should be in the changelog.

- [ ] Add details about auto converting strike to s when using a html5 schema.

Reply:
A changelog entry has been added for this